src/rulesets/Base/base.jl

```diff
@@ -2,62 +2,64 @@
 @scalar_rule(zero(x), Zero())
 @scalar_rule(sign(x), Zero())
 
+@scalar_rule(abs(x::Real), sign(x))
 @scalar_rule(abs2(x), 2x)
+@scalar_rule(exp(x), Ω)
```

Comment:
Not required by Blue Style, but I alphabetised these blocks of code, to make the rules easier to find